package-lock.json

```diff
@@ -4557,6 +4559,22 @@
         "react-dom": "^16.8.0 || ^17.0.0 || ^18.0.0"
       }
     },
+    "node_modules/@storybook/addon-docs/node_modules/@storybook/theming": {
+      "version": "7.6.3",
+      "resolved": "https://associatedpress.jfrog.io/artifactory/api/npm/npm/@storybook/theming/-/theming-7.6.3.tgz",
+      "integrity": "sha512-9ToNU2LM6a2kVBjOXitXEeEOuMurVLhn+uaZO1dJjv8NGnJVYiLwNPwrLsImiUD8/XXNuil972aanBR6+Aj9jw==",
```

Comment:
@csdiehl This PR is using your companies internal NPM registry - can you make the package json changes and install with normal npm?